samples/DevPortal/shared-pom.xml

```diff
@@ -33,6 +33,10 @@
                       path represents the signifies the path to data stored within the info object. 
                         The pipe character can be used to traverse into sub-items. For instance, with a OpenAPI document containing  { "info": { "contact": { "company": "Google" } } }, you could be access the data by setting the path to contact|company.
                       field represents the machine name of the field within the Developer portal.
+
+                      portal.model.config.name can be used to configure the modelName being sent to the SmartDocs API
+                        The caret symbol (^) is used to seperate which fields you want appended to the title from the info object and it will be replaced with _
+                        The pipe character is used to traverse into sub-items if necessary
```

Comment:
Can you provide an example so that its more clear. Like if your config is so and so, the title will be so and so

Reply:
@vuksv 
Reviewed the code:
- Can you please provide an example in the shared-pom as I commented on the file
- Can you please add some info in the [README](https://github.com/apigee/apigee-smartdocs-maven-plugin/tree/master/samples#advanced-implementation-model-metadata-import) on this very specific feature with example

Reply:
I updated the documentation to add an example.